main.py

```
@@ -2,47 +2,52 @@
         mutated = mutate_protocol(protocol)
-        
+
         # Log iteration summary
         stats = get_protocol_stats(protocol)
         if stats:
-            log(f"Current stats - Success rate: {stats['success_rate']:.2%}, "
-                f"Total executions: {stats['total_executions']}")
-        
+            log(
+                f"Current stats - Success rate: {stats['success_rate']:.2%}, "
+                f"Total executions: {stats['total_executions']}"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)